src/hipscat_import/catalog/resume_plan.py

```diff
@@ -249,6 +251,11 @@ def get_alignment_file(
             self.destination_pixel_map = [
                 (order, pix, count) for (order, pix, count) in self.destination_pixel_map if int(count) > 0
             ]
+        total_rows = sum(count for (_, _, count) in self.destination_pixel_map)
+        if total_rows != expected_total_rows:
```

Comment:
Would it fail for the default value (0)?

Reply:
Fair question.

This method doesn't have a default value here. 

In practice, in the pipeline, if they're using the default *argument* of `expected_total_rows=0`, we'll still determine the total rows after the mapping stage. We'll use that value going forward in the pipeline. The call inside run_import.py for `get_alignment_file` passes in the `total_rows` that we got from looking over all of the input files.